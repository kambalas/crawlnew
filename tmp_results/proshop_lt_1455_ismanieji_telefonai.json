[
  {
    "name": "Mobilus Telefonas - Motorola Edge 50 Pro 5G 12/512GB Black Beauty",
    "price": "597,99",
    "error": false
  },
  {
    "name": "Mobilus Telefonas - Motorola ThinkPhone 8GB RAM 256GB Atmintis 6.6\" P-OLED 5G Anglies Juodas",
    "price": "538,99",
    "error": false
  },
  {
    "name": "Mobilus Telefonas - Samsung Galaxy S24 FE 6.7\" 5G Dual SIM Mėlynas",
    "price": "692,99",
    "error": false
  },
  {
    "name": "Mobilus Telefonas - Xiaomi Redmi Note 11S 6/128GB, Švytinčiai Mėlynas",
    "price": "195,09",
    "error": false
  },
  {
    "name": "Mobilus Telefonas - Motorola Edge 50 Neo 5G 12GB 512GB Poinciana Terracotta",
    "price": "534,99",
    "error": false
  },
  {
    "name": "Mobilus Telefonas - Motorola Edge 50 neo 5G 12GB/512GB P-OLED Triple Camera Water-Resistant",
    "price": "534,99",
    "error": false
  },
  {
    "name": "Mobilus Telefonas - Google Pixel 9 5G 12GB RAM 128GB - Obsidian",
    "price": "889,99",
    "error": false
  },
  {
    "name": "Mobilus Telefonas - Motorola Edge 40 Neo, Raminanti Jūra",
    "price": "314,99",
    "error": false
  },
  {
    "name": "Mobilus Telefonas - Google Pixel 9 Pro XL 5G Dual SIM 16/128GB Obsidianas",
    "price": "1184,99",
    "error": false
  },
  {
    "name": "Mobilus Telefonas - Samsung Galaxy A55 5G 6.6\" 8GB 256GB Lilac",
    "price": "521,99",
    "error": false
  },
  {
    "name": "Mobilus Telefonas - Samsung Galaxy Xcover6 Pro Enterprise Edition 16.8 Cm (6.6\") Dual SIM, USB Type-C, 5G 6 GB 128 GB 4050 mAh (Juodas)",
    "price": "541,99",
    "error": false
  },
  {
    "name": "Mobilus Telefonas - Motorola Edge 40 Neo 16,6 cm (6.55\") Dviguba SIM jungtis Android 13 5G C tipo USB 12 GB 256 GB 5000 mAh Mėlyna",
    "price": "320,99",
    "error": false
  }
]