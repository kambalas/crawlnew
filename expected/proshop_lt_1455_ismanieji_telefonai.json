[
  {
    "index": 3,
    "error": true,
    "tags": [
      "error"
    ],
    "content": "'str' object has no attribute 'choices'"
  },
  {
    "name": "Mobilus Telefonas Apple iPhone 16 Pro Max 512GB Desert Titanium",
    "price": "2206,98 €",
    "error": false
  },
  {
    "name": "Apple iphone 14 128gb midnight mpuf3hx/a",
    "price": "784,99 €",
    "error": false
  },
  {
    "name": "Mobilus Telefonas Apple iPhone 16 Pro Max 256GB natūralus titanas",
    "price": "1949,99 €",
    "error": false
  },
  {
    "name": "Mobilus Telefonas Apple iPhone 16 128GB White",
    "price": "1089,99 €",
    "error": false
  },
  {
    "name": "Mobilus Telefonas Apple iPhone 16 Pro Max 512GB Black Titanium",
    "price": "2206,98 €",
    "error": false
  },
  {
    "name": "Mobilus Telefonas Apple iPhone 16 Pro Max 512GB White Titanium",
    "price": "2206,98 €",
    "error": false
  },
  {
    "name": "iPhone 13 128GB (Product) RED",
    "price": "699",
    "error": false
  },
  {
    "name": "iPhone 13 256GB (Product) RED",
    "price": "799",
    "error": false
  },
  {
    "name": "iPhone 12 64GB",
    "price": "599",
    "error": false
  },
  {
    "name": "iPhone 12 128GB",
    "price": "649",
    "error": false
  },
  {
    "name": "iPhone SE 2022 64GB",
    "price": "429",
    "error": false
  },
  {
    "name": "iPhone SE 2022 128GB",
    "price": "479",
    "error": false
  },
  {
    "index": 1,
    "error": true,
    "tags": [
      "error"
    ],
    "content": "'str' object has no attribute 'choices'"
  }
]